packages/teams-js/webpack.config.js

```diff
@@ -27,8 +27,8 @@ module.exports = {
       type: 'umd',
       umdNamedDefine: true,
     },
-    //Prevents 'self' object conflict between nodejs and nextjs
-    //globalObject: 'this',
+    //Typically resolves to 'self' unless running in a server side rendered environment
+    globalObject: "typeof self !== 'undefined' ? self : this",
```

Comment:
Is this something that gets executed at runtime so that globalobject is actually being set when our code is being executed and not when webpack bundles everything?

Reply:
(If so, this is cool and I had no idea you could do this)

Reply:
Yes, that's exactly correct. When webpack creates the minified code for our package, whatever value was given to globalObject was being placed directly in the .min code (previously self or this). Since the value is directly added to the code, we can inject `typeof self !== 'undefined' ? self : this` so the global object gets determined when the package is run.